lib/Wikia/src/Service/User/Attributes/UserAttributes.php

```diff
@@ -68,6 +68,10 @@ public function getAttribute( $userId, $attributeName, $default = null ) {
 
 	private function loadAttributes( $userId ) {
 
+		if ( $userId == 0 ) {
+			return [];
```

Comment:
By returning an empty array here for anonymous users, we prevent an unnecessary call to the attribute service. Those anon users will just use default values for attributes. See the logic [here](https://github.com/Wikia/app/blob/dev/lib/Wikia/src/Service/User/Attributes/UserAttributes.php#L56-L66).